riscv/tests/common/mod.rs

```diff
@@ -56,6 +44,18 @@ pub fn verify_riscv_asm_string<T: FieldElement, S: serde::Serialize + Send + Syn
         );
     }
 
+    // Compute the witness once for all tests that follow.
```

Comment:
moved here to have the fast executor always run before witgen, required by some tests